waku/v2/node/storage/message/waku_message_store.nim

```diff
@@ -23,6 +23,20 @@ type
   WakuMessageStore* = ref object of MessageStore
     database*: SqliteDatabase
 
+proc toBytes(x: float64): seq[byte] =
+  let xbytes =  cast[array[0..7, byte]](x)
+  return @xbytes
+
+proc fromBytes(T: type float64, bytes: seq[byte]): T =
```

Comment:
Hm, any reason we don't use nim-stew/byteutils for this? Seems error-prone

Reply:
Haven't found any support for `float64` in nim-stew/byteutils

Reply:
Might want to create an upstream issue (and keep code local here for now)